scripts/actions/__tests__/fetch-observability-packs.test.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const fs = require('fs');
+const fetch = require('node-fetch');
+const fetchObservabilityPacks = require('../fetch-observability-packs');
+
+jest.mock('node-fetch');
+jest.mock('fs');
+
+const mockAPIResponse = (response) => {
+  fetch.mockResolvedValueOnce({
+    json: jest.fn(() => Promise.resolve(response))
+  });
+};
+
+jest.mock('fs');
+
+describe('Action: Fetch Observability Packs', () => {
+  const fakeAPIURL = 'fakeapi.com/graphql';
+  const fakeToken = 'fake_token';
+  const fakeGqlQuery = 'fake_gql_query';
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+ 
+  test('writes observability packs to file', async () => {
+    const apiReturnValue = {
+      data: {
+        docs: {
+          openInstallation: {
+            observabilityPackSearch: {
+              results: {
+                observabilityPacks: [
+                  {
+                    test: 'test'
+                  }
+                ]
+              }
+            }
+          }
+        }
+      }
+    };
+    fetch.mockResolvedValueOnce({
+      ok: true,
+      json: jest.fn(() => Promise.resolve(apiReturnValue))
+    });
```

Comment:
💯  yay tests!

I _think_ this could be simplified to:

```suggestion
    mockAPIResponse(apiReturnValue);
```

Reply:
Same below.

Reply:
I actually meant to get rid of this function since I need very different response objects for each test! The latest commit removes it.